tmt/schemas/execute/tmt.yaml

```
@@ -23,6 +23,9 @@ properties:
   exit-first:
     type: boolean
 
+  ignore-duration:
```

Just thinking aloud, shouldn't we be more specific and mention `test` as well?
```suggestion
  ignore-test-duration:
```

---

Had a chat on hacking session about this, not accepting on the basis there is no other 'duration'.
We have just 'Test' duration, other 'kill it if it exceeds some time' use term "timeou" (eg reboot, connect...)